Added details of rich text field. (#4301)
umbraco/UmbracoDocs: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md

- **Single Choice**:
    Displays a list of items with a radio button for each item where the user can select a single option.

    ![Single choice](images/singlechoice-v9.png)

- **Title and Description**:

>     Displays a read-only title and description for a set of form fields.

Reviewer: ```suggestion
    Displays a read-only title and description for a set of Form fields.
```

>     Displays a read-only title and description for a set of form fields.

Reviewer: ```suggestion
    Displays a read-only title and description for a set of Form fields.
```

Author: I'd be minded to keep this as "form" as here we are talking about "a form" rather than "Umbraco Forms" the product.  But up to you.

Reviewer: Makes sense.

    ![Title and description](images/titleanddescription-v9.png)

- **Rich Text**:

>     Displays read-only formatted text that can be used to provide additional copy within a form.

Reviewer: ```suggestion
    Displays a read-only formatted text that can be used to provide an additional copy within a Form.
```

Author: As above.  Also, here I don't think we want the "an".  I'm looking to say here you can add additional copy in the sense of "additional formatted text" rather than a copy as in "a duplicate".

Reviewer: Oh ok, now it makes sense to me 😃I was thinking of it as an additional duplicate copy. 

Can we maybe rephrase it to make it more clear? Maybe something like: 
```suggestion
    Displays a read-only tool that can be used to provide additional formatted text within a Form.
```

>     Displays read-only formatted text that can be used to provide additional copy within a form.

Author: Not sure "tool" is right... how about:

```suggestion
    Displays read-only formatted text that can be used to provide additional information and links within a form.
```
